src/main/java/de/doubleslash/keeptime/view/ManageWorkController.java

```diff
@@ -0,0 +1,260 @@
+         final StringProperty text = (StringProperty) observable;
+         try {
+            stringConverter.fromString(newValue);
+            text.setValue(newValue);
+            endTimeSpinner.increment(0); // TODO find better Solution
+         } catch (final DateTimeParseException e) {
+            text.setValue(oldValue);
+         }
+      });
+
+      startTimeSpinner.setValueFactory(new SpinnerValueFactory<LocalTime>() {
+
+         {
+            setConverter(new LocalTimeStringConverter(FormatStyle.MEDIUM));
+         }
+
+         @Override
+         public void decrement(final int steps) {
+            if (getValue() == null) {
+               setValue(LocalTime.now());
+            } else {
+               final LocalTime time = getValue();
+               setValue(time.minusMinutes(steps));
+            }
+
+         }
+
+         @Override
+         public void increment(final int steps) {
+            if (getValue() == null) {
+               setValue(LocalTime.now());
+            } else {
+               final LocalTime time = getValue();
+               setValue(time.plusMinutes(steps));
+            }
+
+         }
+
+      });
+      endTimeSpinner.setValueFactory(new SpinnerValueFactory<LocalTime>() {
+
+         {
+            setConverter(new LocalTimeStringConverter(FormatStyle.MEDIUM));
+         }
+
+         @Override
+         public void decrement(final int steps) {
+            if (getValue() == null) {
+               setValue(LocalTime.now());
+            } else {
+               final LocalTime time = getValue();
+               setValue(time.minusMinutes(steps));
+            }
+
+         }
+
+         @Override
+         public void increment(final int steps) {
+            if (getValue() == null) {
+               setValue(LocalTime.now());
+            } else {
+               final LocalTime time = getValue();
+               setValue(time.plusMinutes(steps));
+            }
+
+         }
+
+      });
+
+      startTimeSpinner.getValueFactory().setValue(work.getStartTime().toLocalTime());
+      endTimeSpinner.getValueFactory().setValue(work.getEndTime().toLocalTime());
+
+      noteTextArea.setText(work.getNotes());
+      projectComboBox.getItems().addAll(model.getAvailableProjects());
+
+      // Dropdown Options
+      projectComboBox.setCellFactory(new Callback<ListView<Project>, ListCell<Project>>() {
+
+         @Override
+         public ListCell<Project> call(final ListView<Project> l) {
+            return new ListCell<Project>() {
+
+               @Override
+               protected void updateItem(final Project item, final boolean empty) {
+                  super.updateItem(item, empty);
+                  if (item == null || empty) {
+                     setGraphic(null);
+                  } else {
+                     final Color color = item.getColor();
+                     final String hexColor = String.format("#%02X%02X%02X", (int) (color.getRed() * 255),
```

Comment:
FYI: Theres a class ColorHelper. Please extract the code here and create a function like colorToHex :)
https://github.com/doubleSlashde/KeepTime/blob/develop/src/main/java/de/doubleslash/keeptime/common/ColorHelper.java

BUT not needed as the logic already exists:
Better use the StyleUtils to change the bg color like in:
https://github.com/doubleSlashde/KeepTime/blob/develop/src/main/java/de/doubleslash/keeptime/view/ViewController.java#L339 
:)